spec/ruby/core/unboundmethod/equal_value_spec.rb

```diff
@@ -98,4 +98,38 @@ def discard_1; :discard; end
 
     (@discard_1 == UnboundMethodSpecs::Methods.instance_method(:discard_1)).should == false
   end
+
+  it "considers methods through aliasing equal" do
```

Comment:
No, because these specs pass for versions < 3.2 due to `method/instance_method` skipping zsuper methods.
I'll add a comment in the file because indeed it is quite subtle.